src/Raven.Studio/typescript/commands/database/documents/saveDocumentCommand.ts

```diff
@@ -5,26 +5,37 @@ import endpoints = require("endpoints");
 
 class saveDocumentCommand extends commandBase {
 
-    constructor(private id: string, private document: document, private db: database, private reportSaveProgress = true) {
+    constructor(private id: string, private document: document, private db: database, private reportSaveProgress: boolean, private forceRevisionCreation: boolean) {
```

Comment:
let's make default value for this parameter = true, as forceRevisionCreation should be rare case when calling this function

Reply:
done